forem/forem
# New home feed display ads API documentation update

## What type of PR is this? (check all applicable)


## Description

This PR is based on top of #19299

This PR finishes the part of the issue #19225 which is updating the API documentation

## Related Tickets & Documents



- Closes #19225

## QA Instructions, Screenshots, Recordings

Execute
`SWAGGER_DRY_RUN=0 RAILS_ENV=test rails rswag PATTERN="spec/requests/api/v1/**/*_spec.rb"`

Then check out `swagger/v1/api_v1.json` on https://editor.swagger.io/

### UI accessibility concerns?

No

## Added/updated tests?
